src/map.rs

```diff
@@ -1325,7 +1346,7 @@ where
     ///
     /// ***Panics*** if `key` is not present in the map.
     fn index(&self, key: &Q) -> &V {
```

Comment:
Just clarifying for future reference -- our `Index` and `IndexMut` implementations don't need `#[track_caller]` because the trait definition already applies it.